data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -345,7 +368,7 @@ In OCaml, _pattern matching_ provides a means to inspect data of any kind, excep
 # let rec sum u =
     match u with
     | [] -> 0
-    | x :: v -> x + sum v;;
+    | x :: v -> x + sum v;; (* :: in a pattern is not the cons operator *)
```

Comment:
What do you mean here?

Reply:
`1 :: [2]` is the cons operator, right? The double colon is used in `match` branches but it's not actually the cons operator, it's just `match`-specific syntax that looks the same: `| hd :: tl -> ...`. Unless I'm mistaken, you can't write `| (::)(hd, tl) -> ...`. That's what I'm trying to communicate.

Reply:
```ocaml
List.(function ([]) -> None | (::) (x, u) -> Some (x, u))
```

Reply:
👀 okay what the hell. I thought I tried this but it seems I miswrote it or something cuz now it works as you demonstrated. Folks on the discord explained it, so I'll remove this comment.

Reply:
Removed!